.gitignore

```diff
@@ -21,3 +21,4 @@ internal/venice-test-common/src/jmh/generated
 _site/
 Gemfile.lock
 .bundles_cache
+/.ciconfig
```

Comment:
This file should be in your home folder, so I don't think we need this unless you put this file in the checkout dir of `venice`, right?

Reply:
I didn't know that and it's currently in my checkout dir :sweat_smile: That's why I wanted to exclude it. I can remove this line if we think there's a chance we will check in this file to the repo

Reply:
We can checkin this file since it is internal.
Just put this file in your home dir, and you are good to run the internal ci.

Reply:
I see, thanks.

Reply:
Removed.